### DISCUSSION TITLE
error running Prisma in AWS Lambda/SAM. ENOENT /var/task/schema.prisma

### DISCUSSION THREAD

**Discussion by carlosLMZarego** [prisma/prisma#18837] [Q&A]
Hello everybody, I'm trying to run some Prisma queries in my lambda, I ran my migration and I initialized my PrismaClient

```ts
const prisma = new PrismaClient()
//some lines here
const organization = await prisma.organization.create({
  data : organizationRequest
})

```

the autocomplete works smoothly and everything seems ok, but when I run my code I get this error

```
 {
    "errno": -2,
    "syscall": "open",
    "code": "ENOENT",
    "path": "/var/task/schema.prisma",
    "clientVersion": "4.12.0"
  }
```

I don't know why prisma is trying to find the schema in that path...also, my lambda DOES NOT HAVE access to any folder because all files are compiled to a single file using esbuild, I could use esbuild in order to get the .prisma file and pass to the client like this

```ts
import schema from "./prisma/schema.prisma"
const prisma = new PrismaClient({schema})
```

My question are 
1)how you can pass a custom schema.prisma route to my PrismaClient?
2) or do you know another possible solution for this problem?

thank you so much

**Comment by willruggiano** [comment]
You need to copy the schema.prisma file to the root of the Lambda deployment archive. Depends how you are bundling your Lambda code. For example, if you are using the AWS CDK construct NodejsFunction you can add a `beforeInstall` commandHook that just copies the schema file to the outputDir

**Reply by nurul3101** [reply]
@carlosLMZarego Did you had a chance to look at will's response? Do let us know in case you need any assistance.
